src/Serializer/DataArraySerializer.php

```diff
@@ -0,0 +1,29 @@
+<?php namespace League\Fractal\Serializer;
+
+class DataArraySerializer extends ArraySerializer
+{
+
+    /**
+     * Serialize the top level data.
+     * 
+     * @param array $data
+     * 
+     * @return array
+     */
+    public function serializeData(array $data)
+    {
+        return array('data' => $data);
+    }
+
+    /**
+     * Serialize the top level meta data.
+     * 
+     * @param array $meta
+     * 
+     * @return array
+     */
+    public function serializeMeta(array $meta)
```

Comment:
@philsturgeon I am trying to set meta in my transformer for included relationship but when i output my collection it wont return meta but if i set resource key it sets the resource key.. What is the problem ?? I cant get it. If you can help me out